tox.ini

```diff
@@ -1,17 +1,19 @@
 [tox]
-envlist = py{34,35,36}-djangomaster, py{27,34,35,36}-django{18,19,110,111}, py33-django18, lint, docs
+envlist = py{34,35,36}-djangomaster, py{27}-django{18,19,110,111}, py{34,35,36}-django{18,19,110,111,200}, py33-django18, lint, docs
```

Comment:
this should only be supported django and python versions. eg py27,34,35,36 and django18,111,200

`py{27}-django{18,111}, py{34,35,36}-django{18,111,200}`

Reply:
I was thinking about clipping the matrix, but they where already here and they pass the tests. 

If you want I can remove those unsupported versions from here, but I don't see the harm in leaving it as it is.

Reply:
always remove unsupported python and django versions. Otherwise people will try to make their tests pass in the old versions when they submit changes.